demo/generate-features/README.md

```
@@ -56,17 +67,112 @@ features:
+                        default: "0.115"
+                ignore:
+                    - '.*flask.*'
 ```
 
+## Multi-Valued Feature (prompt_mode: select)
+
+The `web_framework` feature uses the multi-valued `prompt_mode: select`
+schema. Under the hood it delegates to `cement.utils.shell.Prompt` with
+`numbered=True`, so the user sees a numbered picker:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)